.github/workflows/ci.yml

```diff
@@ -0,0 +1,62 @@
+name: CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  CI:
+    runs-on: ubuntu-18.04
```

Comment:
can we also try on 20.04?

Reply:
the actual container is what runs here... to build/test not the runner..

But lets change it to ubuntu-latest maybe?

Reply:
There's some reason why ubuntu 20 breaks our docker builds - we can revisit this later on repos that matter.